crates/eth-rpc/src/servers/alchemy_rpc.rs

```diff
@@ -22,8 +30,117 @@ impl<P: Provider + Send + Sync> AlchemyRpc<P> {
 
 #[async_trait]
 impl<P: Provider + Send + Sync + 'static> AlchemyApiServer for AlchemyRpc<P> {
+    async fn token_allowance(
+        &self,
+        contract_address: Address,
+        account_address: Address,
+        spender_address: Address,
+    ) -> Result<TokenAllowance> {
+        let entry_point: Felt252Wrapper = keccak256("allowance(account, spender)")
+            .try_into()
+            .map_err(|e: ConversionError<()>| EthApiError::<P::Error>::ConversionError(e.to_string()))?;
```

Comment:
Could this work?
`.map_err(EthApiError::<P::Error>::from)`

Reply:
That works 👍